pdfjs.config

```
@@ -1,5 +1,5 @@
 {
   "stableVersion": "4.4.168",
-  "baseVersion": "fdb3617e0f7ebbefce9a65a87971251e92653d07",
-  "versionPrefix": "4.4."
+  "baseVersion": "64635f3b35a64d1f941aadaf2bdcfd114b7053cc",
```

Generally we reference the merge-commit here, since that's where it landed in the repository.
```suggestion
  "baseVersion": "bdcc4a0febc02aec5be79da67507972e3c7be280",
```